doc/nrf/gs_dtm_testing.rst

```diff
@@ -0,0 +1,118 @@
+.. _gs_dtm_testing:
+
+Direct Test Mode terminal connection
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Follow the instructions in this section to connect with the DK which is running the DTM to send commands and receive responses.
```

Comment:
To run Direct Test Mode (DTM) to send commands and receive responses, you can connect to your development kit with RealTerm in Windows or Minicom in Linux.